src/main/java/frc/robot/commands/drivetrain/LowSpeed.java

```diff
@@ -0,0 +1,65 @@
+/* ==================================================
+ * Authors: Lucas Jacobs
+ *
+ * --------------------------------------------------
+ * Description: Sets the Drivetrain to low speed
+ * ================================================== */
+
+package frc.robot.commands.drivetrain;
+
+import frc.robot.Robot;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class LowSpeed extends CommandBase
+{
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+
+  // VARIABLES //
+
+  private Robot robot;
+
+  // CONSTANTS //
+
+
+
+  // CONSTRUCTOR //
+
+  public LowSpeed(Robot robot)
+  {
+    this.robot = robot;
+
+    // subsystems that this command requires
+    addRequirements(robot.drivetrain);
```

Comment:
This command doesn't use the drivetrain at all, it's just used to change the settings for the driver.  Maybe we can make a new command folder for driver and operators commands and put this there.  It's not really related to the drivetrain.

Reply:
Also might be a good idea to make a `teleop` folder under commands and just put all the teleop subsystem command folders under there to keep things organized.

Reply:
I got rid of the drivetrain requirement, but I don't think we need a teleop folder. All the auto commands can go in one folder, and adding teleop means changing the packages and imports in like 20 different files.

Reply:
That's true, it adding a teleop folder will just screw up all the imports so best to leave it.